### PR TITLE
Make HTML-like language syntax highlighting consistent

### PR DESCRIPTION
**Pull request by Huliiiiii**
Changes:
- tag name: using `tag`
- doctype: using `tag`
- attribute name: using `attribute`
- angle bracket: using `punctuation.bracket`
- `@tag.delimiter` => `@punctuation.delimiter` https://github.com/zed-industries/zed/blob/0fd50302979539fbcb00cac7b29fd9bfe434b4f5/docs/src/extensions/languages.md?plain=1#L117

Release Notes:

- N/A

### CODE REVIEW

File: extensions/astro/languages/astro/highlights.scm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (tag_name) @tag
 (erroneous_end_tag_name) @keyword
-(doctype) @constant
-(attribute_name) @property
+(doctype) @tag
```
Comment (maxdeviant) [inline extensions/astro/languages/astro/highlights.scm:3]: The use of `@constant` for the doctype is intentional, as we want it to be visually distinct from the other tags.

Comment (maxdeviant) [reply extensions/astro/languages/astro/highlights.scm:3]: If we do want to change it we could do something like `@tag.doctype` to allow themes to individually target it, if desired.

File: extensions/astro/languages/astro/highlights.scm [hunk 2]
```diff
@@ -22,4 +22,4 @@
   ">"
   "</"
   "/>"
-] @tag.delimiter
+] @punctuation.bracket
```
Comment (Huliiiiii) [inline extensions/astro/languages/astro/highlights.scm:25]: Would it be better to use `punctuation.tag` or `punctuation.bracket.tag` or something like that?

**Comment by maxdeviant** [comment]
I'm going to close this out, as some of the changes have already been applied and the Vue extension has been moved to a separate repo.

**Pull request closed without merging**